sensu/plugins/metrics-net.rb

```diff
@@ -80,7 +85,7 @@ def run
     Dir.glob('/sys/class/net/*').each do |iface_path|
       next if File.file?(iface_path)
       iface = File.basename(iface_path)
-      next if iface == 'lo'
```

Comment:
We might want to filter out 'lo' even when the all flag is set, depending on the requirements